data/canonical-tags/architecture.yaml

```diff
@@ -1,3 +1,5 @@
 id: architecture
 name: Architecture
 description: The inner components of Kubernetes.
+nameKey: canonical_tag_architecture_name
+descriptionKey: canonical_tag_architecture_description
```

Comment:
Could we use the `id` here (`"architecture"`)?

Reply:
I think so. I played a bit around with Hugo trying to interpolate the id but wasn't super successful. I'll give it another try.

Reply:
I was able to make it work 🎉